lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.0-beta.6",
+  "version": "1.0.0-beta-json.7",
```

Comment:
Don't worry about the versioning. I will handle it from my side. However I'm curious about why you called it `-beta-json.7`???